incubator/cassandra/sample/create-storage-gce.yaml

```diff
@@ -0,0 +1,7 @@
+kind: StorageClass
```

Comment:
In order for the chart to be portable, it should just rely on requesting a PVC and not be creating any PVs or Storage Classes. More info here https://kubernetes.io/docs/user-guide/persistent-volumes/#writing-portable-configuration